interfaces/Config/templates/config_switches.tmpl

```diff
@@ -324,11 +324,6 @@
                         <span class="desc">$T('explain-rating_enable')</span>
```

Comment:
With the changes in 1.1.0 we now have more indexers in the wild actually use the system, by sending the headers or using the fields in the NZB. 
But nowadays very few files have actual audio/video ratings of users, so the main purpose of this feature has evolved to automatically report back if the download failed on the server.

The text is also updated as such, that enabling it will send to the indexer if a download failed, only if it failed.